src/content/structured/components/popover-menu/code.mdx

```diff
@@ -246,10 +256,16 @@ export const snippetsButtons = [
 </div>`,
       long: `{shortCode}
 <script>
-  var icPopover = document.querySelector("#popover-menu-2");
-  function handleClick() {
-    icPopover.open = !icPopover.open;
-  }
+    var icPopover = document.querySelector("#popover-menu-2");
+    var anchorButton = document.querySelector("#button-2");
+    function handleClick() {
+      icPopover.open = !icPopover.open;
+      anchorButton.setAttribute('aria-expanded', icPopover.open);
+    }
+    icPopover.addEventListener('icPopoverClosed', handleClose);
+    function handleClose() {
+      anchorButton.setAttribute('aria-expanded', 'false');
+    }
```

Comment:
Please reduce the white space before each line so it has the same whitespace as before